tests/baselines/reference/parserRealSource7.errors.txt

```diff
@@ -1562,6 +1563,8 @@ tests/cases/conformance/parser/ecmascript5/parserRealSource7.ts(828,13): error T
                     fgSym.type &&
                     fgSym.type.construct &&
                     fgSym.type.construct.signatures != [] &&

```

Comment:
wat

Reply:
I don't know where is this source comes from 😂